setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+#Script to run To-do app using the bash script
+
+#Step1: Download package from the repository  
+sudo apt-get update -y
+
+#Step2: Install PyEnv Requisites using apt get
+sudo apt-get install -y git
+
+#Step3: Installs pip Python3 Version 
+sudo apt-get install python3-pip
+
+#Step4: Copy poetry installation file to local 
+curl -sSL https://raw.githubusercontent.com/python-poetry/ \
+poetry/master/get-poetry.py | python
+
+#Step5: Clone repository of To-Do App code from GIT
+git clone https://github.com/sunnygunjesh/DevOps-Course-Starter.git
+
+#Step6:Install poetry
+poetry install 
```

Comment:
If poetry has just been installed it likely isn't on your Path yet so this will fail; you can either manually set the default environment variables:
```bash
export POETRY_ROOT=\"$HOME//.poetry/\"
export PATH=\"$POETRY_ROOT/bin:$PATH\"
```
or you could source the ones that it should have added to your profile, e.g. after installing poetry, run:
```bash
source ~/.profile
```